2025R2/dpf-operator-spec-25-r2/core-concepts/dpf-types.md

```diff
@@ -132,48 +132,133 @@ For cyclic simulation models, the ``cyclic_support`` holds information about the
 
 ### Filtering
 
+<a id="filtering"></a>
+
+When handling large amounts of simulation data, you can select a subset of the data and filter it down to the data of interest to the analysis.
+
+In the DPF ecosystem this is called ``scoping`` the data, and is managed with ``scoping`` entities.
+
+#### Scoping
+
 <a id="scoping"></a>
 
-- scoping
+A ``scoping`` describes a typed list of IDs. 
+
+It allows you to select and filter data when given as input to operators, or data description when produced as output.
+
+Each ``field`` data storage type has a ``scoping`` associated to it, describing the subset of its ``support`` that the data applies to. The data in a ``field`` is ordered the same way as the IDs in the ``scoping``, allowing you to reorder the data.
```

Comment:
I'd eliminate reference to reordering, as here we are describing data, not how we operate with it

Reply:
But the fact that the scoping gives the ordering of the data of the field in terms of entity IDs is a description of the data model, no?
At least that was the objective of this sentence: to explain that the ordering of the scoping is the one to look at. I still get too many questions regarding that point, so I thought I'd mention it here too.

Reply:
Yes, I was referring to the last sentence, I'd eliminate `allowing you to reorder the data.`